### DISCUSSION TITLE
Creating a TIF with a Layer Mask from two files

### DISCUSSION THREAD

**Discussion by edenaj** [ImageMagick/ImageMagick#5670] [Digital Image Processing]
Hi
I could do with some help i am looking to use a automated ai for cutouts of objects to be used in Photoshop, i use TIFs as my main image format and a none distructive Layer Mask for items cutout.
The website i am looking to use for some automation supplies back a color.jpg (original image) and a alpha.png that has the mask. when i use the below
convert color.jpg alpha.png -alpha off -compose CopyOpacity -composite color2.tif
It gives me a tif with the mask applied on a transparent Layer which is great but i need the mask to be none destructive so i can refine the mask should i need to. 
Is there any way of applying the alpha.png file as a Layer Mask

**Comment by hauke28** [comment]
Not exactly what was asked here, but the following command creates a tiff with an editable alpha *channel* (not layer mask). Informatin in background pixel is invisible, but not deleted and can be recovered.
`magick color.jpg alpha.png -alpha off -channel-fx "| gray=>alpha" -define tiff:alpha=unspecified color2.tif`
